kythe/proto/analysis.proto

```diff
@@ -211,6 +212,13 @@ message KzipInfo {
   repeated string critical_kzip_errors = 5;
 }
 
+// RepoMetadata provides storage for additional information about a specific
+// build from a repository. This message is stored in the
+// CompilationUnit.details field.
+message BuildMetadata {
+  google.protobuf.Timestamp commit_timestamp = 1;
```

Comment:
Let's document that this is reasonably used for the "primary repository" commit, but could just as easily be some other origin timestamp.